packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -160,11 +178,24 @@ class DropdownList extends React.Component {
    * @return {Array<React.ReactNode>} the array of children
    */
   cloneChildren() {
-    return React.Children.map(this.props.children, (child, index) => React.cloneElement(child, {
-      isActive: index === this.state.active,
-      requestClose: this.props.requestClose,
-      'data-terra-dropdown-list-item': true,
-    }));
+    return React.Children.map(this.props.children, (child, index) => {
+      const currentItemLabel = this.props.children[index]?.props.label;
+      const currentIndex = index + 1;
+      const ofText = this.props.intl.formatMessage({ id: 'Terra.dropdownButton.of' });
+      const totalItems = this.props.children.length;
+      let ariaLabel = null;
+      if (currentIndex === 1 && totalItems) {
+        ariaLabel = `${this.expanded}${currentItemLabel},(${currentIndex} ${ofText} ${totalItems})`;
```

Comment:
Instead of building the phrase ourselves, we should make this a single string so that it can be translated within the context of the usage. e.g.
- [terra-form-select (line 9)](https://github.com/cerner/terra-core/blob/0d0fe3c8865823913787166bee9047ef815ddc34/packages/terra-form-select/translations/en.json#LL9C5-L9C5)
- [terra-paginator (line 7)](https://github.com/cerner/terra-core/blob/0d0fe3c8865823913787166bee9047ef815ddc34/packages/terra-paginator/translations/en.json#L7)

Reply:
Implemented as suggested Thanks.